Project is not buildable with Java 17

## Describe the bug
I tried to build the project with Java 17 and got a failure.

### Screenshots/Error Messages
<details>
  <summary>Click to expand the build log!</summary>

  ```log
[INFO] --- spotless-maven-plugin:2.22.2:check (spotless-check) @ business-partner-validation ---
[ERROR] Step 'google-java-format' found problem in 'src/test/java/net/catenax/edc/validation/businesspartner/BusinessPartnerValidationExtensionTest.java':
You are not using latest version on JVM 11+.
Try to upgrade to google-java-format 1.13.0, which may have fixed this problem.
java.lang.Exception: You are not using latest version on JVM 11+.
Try to upgrade to google-java-format 1.13.0, which may have fixed this problem.
    at com.diffplug.spotless.Jvm$Support$1.apply (Jvm.java:181)
    at com.diffplug.spotless.FormatterStepImpl$Standard.format (FormatterStepImpl.java:78)
    at com.diffplug.spotless.FormatterStep$Strict.format (FormatterStep.java:88)
    at com.diffplug.spotless.Formatter.compute (Formatter.java:230)
    at com.diffplug.spotless.PaddedCell.calculateDirtyState (PaddedCell.java:203)
    at com.diffplug.spotless.PaddedCell.calculateDirtyState (PaddedCell.java:190)
    at com.diffplug.spotless.maven.SpotlessCheckMojo.process (SpotlessCheckMojo.java:51)
    at com.diffplug.spotless.maven.AbstractSpotlessMojo.execute (AbstractSpotlessMojo.java:195)
    at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo (DefaultBuildPluginManager.java:137)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:210)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:156)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:148)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:117)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:81)
    at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build (SingleThreadedBuilder.java:56)
    at org.apache.maven.lifecycle.internal.LifecycleStarter.execute (LifecycleStarter.java:128)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:305)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:192)
    at org.apache.maven.DefaultMaven.execute (DefaultMaven.java:105)
    at org.apache.maven.cli.MavenCli.execute (MavenCli.java:957)
    at org.apache.maven.cli.MavenCli.doMain (MavenCli.java:289)
    at org.apache.maven.cli.MavenCli.main (MavenCli.java:193)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0 (Native Method)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke (NativeMethodAccessorImpl.java:77)
    at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke (DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke (Method.java:568)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced (Launcher.java:282)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launch (Launcher.java:225)
    at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode (Launcher.java:406)
    at org.codehaus.plexus.classworlds.launcher.Launcher.main (Launcher.java:347)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0 (Native Method)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke (NativeMethodAccessorImpl.java:77)
    at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke (DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke (Method.java:568)
    at org.apache.maven.wrapper.BootstrapMainStarter.start (BootstrapMainStarter.java:47)
    at org.apache.maven.wrapper.WrapperExecutor.execute (WrapperExecutor.java:156)
    at org.apache.maven.wrapper.MavenWrapperMain.main (MavenWrapperMain.java:72)
Caused by: java.lang.reflect.InvocationTargetException
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0 (Native Method)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke (NativeMethodAccessorImpl.java:77)
    at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke (DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke (Method.java:568)
    at com.diffplug.spotless.java.GoogleJavaFormatStep$State.lambda$createFormat$1 (GoogleJavaFormatStep.java:176)
    at com.diffplug.spotless.FormatterFunc.apply (FormatterFunc.java:32)
    at com.diffplug.spotless.Jvm$Support$1.apply (Jvm.java:179)
    at com.diffplug.spotless.FormatterStepImpl$Standard.format (FormatterStepImpl.java:78)
    at com.diffplug.spotless.FormatterStep$Strict.format (FormatterStep.java:88)
    at com.diffplug.spotless.Formatter.compute (Formatter.java:230)
    at com.diffplug.spotless.PaddedCell.calculateDirtyState (PaddedCell.java:203)
    at com.diffplug.spotless.PaddedCell.calculateDirtyState (PaddedCell.java:190)
    at com.diffplug.spotless.maven.SpotlessCheckMojo.process (SpotlessCheckMojo.java:51)
    at com.diffplug.spotless.maven.AbstractSpotlessMojo.execute (AbstractSpotlessMojo.java:195)
    at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo (DefaultBuildPluginManager.java:137)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:210)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:156)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:148)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:117)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:81)
    at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build (SingleThreadedBuilder.java:56)
    at org.apache.maven.lifecycle.internal.LifecycleStarter.execute (LifecycleStarter.java:128)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:305)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:192)
    at org.apache.maven.DefaultMaven.execute (DefaultMaven.java:105)
    at org.apache.maven.cli.MavenCli.execute (MavenCli.java:957)
    at org.apache.maven.cli.MavenCli.doMain (MavenCli.java:289)
    at org.apache.maven.cli.MavenCli.main (MavenCli.java:193)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0 (Native Method)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke (NativeMethodAccessorImpl.java:77)
    at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke (DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke (Method.java:568)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced (Launcher.java:282)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launch (Launcher.java:225)
    at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode (Launcher.java:406)
    at org.codehaus.plexus.classworlds.launcher.Launcher.main (Launcher.java:347)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0 (Native Method)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke (NativeMethodAccessorImpl.java:77)
    at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke (DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke (Method.java:568)
    at org.apache.maven.wrapper.BootstrapMainStarter.start (BootstrapMainStarter.java:47)
    at org.apache.maven.wrapper.WrapperExecutor.execute (WrapperExecutor.java:156)
    at org.apache.maven.wrapper.MavenWrapperMain.main (MavenWrapperMain.java:72)
Caused by: java.lang.IllegalAccessError: class com.google.googlejavaformat.java.JavaInput (in unnamed module @0x4d705112) cannot access class com.sun.tools.javac.parser.Tokens$TokenKind (in module jdk.compiler) because module jdk.compiler does not export com.sun.tools.javac.parser to unnamed module @0x4d705112
    at com.google.googlejavaformat.java.JavaInput.buildToks (JavaInput.java:349)
    at com.google.googlejavaformat.java.JavaInput.buildToks (JavaInput.java:334)
    at com.google.googlejavaformat.java.JavaInput.<init> (JavaInput.java:276)
    at com.google.googlejavaformat.java.Formatter.getFormatReplacements (Formatter.java:280)
    at com.google.googlejavaformat.java.Formatter.formatSource (Formatter.java:267)
    at com.google.googlejavaformat.java.Formatter.formatSource (Formatter.java:233)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0 (Native Method)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke (NativeMethodAccessorImpl.java:77)
    at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke (DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke (Method.java:568)
    at com.diffplug.spotless.java.GoogleJavaFormatStep$State.lambda$createFormat$1 (GoogleJavaFormatStep.java:176)
    at com.diffplug.spotless.FormatterFunc.apply (FormatterFunc.java:32)
    at com.diffplug.spotless.Jvm$Support$1.apply (Jvm.java:179)
    at com.diffplug.spotless.FormatterStepImpl$Standard.format (FormatterStepImpl.java:78)
    at com.diffplug.spotless.FormatterStep$Strict.format (FormatterStep.java:88)
    at com.diffplug.spotless.Formatter.compute (Formatter.java:230)
    at com.diffplug.spotless.PaddedCell.calculateDirtyState (PaddedCell.java:203)
    at com.diffplug.spotless.PaddedCell.calculateDirtyState (PaddedCell.java:190)
    at com.diffplug.spotless.maven.SpotlessCheckMojo.process (SpotlessCheckMojo.java:51)
    at com.diffplug.spotless.maven.AbstractSpotlessMojo.execute (AbstractSpotlessMojo.java:195)
    at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo (DefaultBuildPluginManager.java:137)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:210)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:156)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:148)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:117)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:81)
    at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build (SingleThreadedBuilder.java:56)
    at org.apache.maven.lifecycle.internal.LifecycleStarter.execute (LifecycleStarter.java:128)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:305)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:192)
    at org.apache.maven.DefaultMaven.execute (DefaultMaven.java:105)
    at org.apache.maven.cli.MavenCli.execute (MavenCli.java:957)
    at org.apache.maven.cli.MavenCli.doMain (MavenCli.java:289)
    at org.apache.maven.cli.MavenCli.main (MavenCli.java:193)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0 (Native Method)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke (NativeMethodAccessorImpl.java:77)
    at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke (DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke (Method.java:568)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced (Launcher.java:282)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launch (Launcher.java:225)
    at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode (Launcher.java:406)
    at org.codehaus.plexus.classworlds.launcher.Launcher.main (Launcher.java:347)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0 (Native Method)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke (NativeMethodAccessorImpl.java:77)
    at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke (DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke (Method.java:568)
    at org.apache.maven.wrapper.BootstrapMainStarter.start (BootstrapMainStarter.java:47)
    at org.apache.maven.wrapper.WrapperExecutor.execute (WrapperExecutor.java:156)
    at org.apache.maven.wrapper.MavenWrapperMain.main (MavenWrapperMain.java:72)
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary for product-edc-parent 0.0.2-SNAPSHOT:
[INFO]
[INFO] product-edc-parent ................................. SUCCESS [  0.536 s]
[INFO] edc-extensions ..................................... SUCCESS [  0.068 s]
[INFO] business-partner-validation ........................ FAILURE [  1.877 s]
[INFO] aas-controller ..................................... SKIPPED
[INFO] postgresql-migration ............................... SKIPPED
[INFO] edc-controlplane-cosmosdb .......................... SKIPPED
[INFO] edc-controlplane-memory ............................ SKIPPED
[INFO] edc-controlplane-postgresql ........................ SKIPPED
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  2.585 s
[INFO] Finished at: 2022-04-27T14:31:26+02:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal com.diffplug.spotless:spotless-maven-plugin:2.22.2:check (spotless-check) on project business-partner-validation: Execution spotless-check of goal com.diffplug.spotless:spotless-maven-plugin:2.22.2:check failed: java.lang.Exception: You are not using latest version on JVM 11+.
[ERROR] Try to upgrade to google-java-format 1.13.0, which may have fixed this problem.: InvocationTargetException: class com.google.googlejavaformat.java.JavaInput (in unnamed module @0x4d705112) cannot access class com.sun.tools.javac.parser.Tokens$TokenKind (in module jdk.compiler) because module jdk.compiler does not export com.sun.tools.javac.parser to unnamed module @0x4d705112
[ERROR] -> [Help 1]
[ERROR]
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR]
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/PluginExecutionException
[ERROR]
[ERROR] After correcting the problems, you can resume the build with the command
[ERROR]   mvn <args> -rf :business-partner-validation
  ```
</details>

## Context Informations
_Add any other context about the probleme here._

- Used version: 96204d0d267402f4be50305e85f9b65870b3f5e1
- OS: Mac OS (M1)
- Java Version:
```plain
openjdk 17.0.2 2022-01-18 LTS
OpenJDK Runtime Environment Zulu17.32+13-CA (build 17.0.2+8-LTS)
OpenJDK 64-Bit Server VM Zulu17.32+13-CA (build 17.0.2+8-LTS, mixed mode, sharing)
```

--- a/.mvn/jvm.config
+++ b/.mvn/jvm.config
@@ -1,0 +1,1 @@
+--add-exports jdk.compiler/com.sun.tools.javac.api=ALL-UNNAMED --add-exports jdk.compiler/com.sun.tools.javac.file=ALL-UNNAMED --add-exports jdk.compiler/com.sun.tools.javac.parser=ALL-UNNAMED --add-exports jdk.compiler/com.sun.tools.javac.tree=ALL-UNNAMED --add-exports jdk.compiler/com.sun.tools.javac.util=ALL-UNNAMED

--- a/edc-extensions/business-partner-validation/pom.xml
+++ b/edc-extensions/business-partner-validation/pom.xml
@@ -67,23 +67,13 @@
         <!-- Test Dependencies -->
         <dependency>
             <groupId>org.junit.jupiter</groupId>
-            <artifactId>junit-jupiter-engine</artifactId>
-        </dependency>
-        <dependency>
-            <groupId>org.junit.jupiter</groupId>
-            <artifactId>junit-jupiter-api</artifactId>
-        </dependency>
-        <dependency>
-            <groupId>org.junit.jupiter</groupId>
-            <artifactId>junit-jupiter-params</artifactId>
-        </dependency>
-        <dependency>
-            <groupId>org.junit.platform</groupId>
-            <artifactId>junit-platform-suite</artifactId>
+            <artifactId>junit-jupiter</artifactId>
+            <scope>test</scope>
         </dependency>
         <dependency>
             <groupId>org.mockito</groupId>
-            <artifactId>mockito-all</artifactId>
+            <artifactId>mockito-core</artifactId>
+            <scope>test</scope>
         </dependency>
     </dependencies>
 </project>

--- a/edc-extensions/business-partner-validation/src/test/java/net/catenax/edc/validation/businesspartner/BusinessPartnerValidationExtensionTest.java
+++ b/edc-extensions/business-partner-validation/src/test/java/net/catenax/edc/validation/businesspartner/BusinessPartnerValidationExtensionTest.java
@@ -25,20 +25,19 @@ import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
 import org.mockito.Mockito;
 
-public class BusinessPartnerValidationExtensionTest {
+class BusinessPartnerValidationExtensionTest {
 
   private BusinessPartnerValidationExtension extension;
 
   // mocks
   private ServiceExtensionContext serviceExtensionContext;
   private PolicyEngine policyEngine;
-  private RuleBindingRegistry ruleBindingRegistry;
 
   @BeforeEach
-  public void setup() {
+  void setup() {
 
     policyEngine = Mockito.mock(PolicyEngine.class);
-    ruleBindingRegistry = Mockito.mock(RuleBindingRegistry.class);
+    RuleBindingRegistry ruleBindingRegistry = Mockito.mock(RuleBindingRegistry.class);
 
     final Monitor monitor = Mockito.mock(Monitor.class);
     serviceExtensionContext = Mockito.mock(ServiceExtensionContext.class);
@@ -52,7 +51,7 @@ public class BusinessPartnerValidationExtensionTest {
   }
 
   @Test
-  public void testRegisterDutyFunction() {
+  void testRegisterDutyFunction() {
 
     // invoke
     extension.initialize(serviceExtensionContext);
@@ -67,7 +66,7 @@ public class BusinessPartnerValidationExtensionTest {
   }
 
   @Test
-  public void testRegisterPermissionFunction() {
+  void testRegisterPermissionFunction() {
 
     // invoke
     extension.initialize(serviceExtensionContext);
@@ -82,7 +81,7 @@ public class BusinessPartnerValidationExtensionTest {
   }
 
   @Test
-  public void testRegisterProhibitionFunction() {
+  void testRegisterProhibitionFunction() {
 
     // invoke
     extension.initialize(serviceExtensionContext);

--- a/edc-extensions/business-partner-validation/src/test/java/net/catenax/edc/validation/businesspartner/functions/AbstractBusinessPartnerValidationTest.java
+++ b/edc-extensions/business-partner-validation/src/test/java/net/catenax/edc/validation/businesspartner/functions/AbstractBusinessPartnerValidationTest.java
@@ -27,7 +27,7 @@ import org.junit.jupiter.params.ParameterizedTest;
 import org.junit.jupiter.params.provider.EnumSource;
 import org.mockito.Mockito;
 
-public class AbstractBusinessPartnerValidationTest {
+class AbstractBusinessPartnerValidationTest {
 
   private AbstractBusinessPartnerValidation validation;
 
@@ -37,7 +37,7 @@ public class AbstractBusinessPartnerValidationTest {
   private ParticipantAgent participantAgent;
 
   @BeforeEach
-  public void BeforeEach() {
+  void BeforeEach() {
     this.monitor = Mockito.mock(Monitor.class);
     this.policyContext = Mockito.mock(PolicyContext.class);
     this.participantAgent = Mockito.mock(ParticipantAgent.class);
@@ -49,7 +49,7 @@ public class AbstractBusinessPartnerValidationTest {
 
   @ParameterizedTest
   @EnumSource(Operator.class)
-  public void testThrowsOnUnsupportedOperations(Operator operator) {
+  void testThrowsOnUnsupportedOperations(Operator operator) {
 
     if (operator == Operator.EQ) { // only allowed operator
       return;
@@ -66,7 +66,7 @@ public class AbstractBusinessPartnerValidationTest {
   }
 
   @Test
-  public void testThrowsOnUnsupportedRightValue() {
+  void testThrowsOnUnsupportedRightValue() {
 
     // prepare
     prepareContextProblems(null);
@@ -82,7 +82,7 @@ public class AbstractBusinessPartnerValidationTest {
   }
 
   @Test
-  public void testValidationFailsWhenClaimMissing() {
+  void testValidationFailsWhenClaimMissing() {
 
     // prepare
     prepareContextProblems(null);
@@ -95,7 +95,7 @@ public class AbstractBusinessPartnerValidationTest {
   }
 
   @Test
-  public void testValidationSuccedesWhenClaimContainsNumber() {
+  void testValidationSuccedesWhenClaimContainsNumber() {
 
     // prepare
     prepareContextProblems(null);
@@ -114,7 +114,7 @@ public class AbstractBusinessPartnerValidationTest {
   }
 
   @Test
-  public void testValidationWhenParticipantHasProblems() {
+  void testValidationWhenParticipantHasProblems() {
 
     // prepare
     prepareContextProblems(Collections.singletonList("big problem"));
@@ -128,7 +128,7 @@ public class AbstractBusinessPartnerValidationTest {
   }
 
   @Test
-  public void testValidationWhenParticipantIsValid() {
+  void testValidationWhenParticipantIsValid() {
 
     // prepare
     prepareContextProblems(null);

--- a/edc-extensions/pom.xml
+++ b/edc-extensions/pom.xml
@@ -26,8 +26,8 @@
     <packaging>pom</packaging>
 
     <modules>
-        <module>business-partner-validation</module>
         <module>aas-controller</module>
+        <module>business-partner-validation</module>
         <module>postgresql-migration</module>
     </modules>
 

--- a/pom.xml
+++ b/pom.xml
@@ -61,7 +61,7 @@
         <!-- test dependency version -->
         <junit.jupiter.version>5.8.2</junit.jupiter.version>
         <junit.platform.version>1.8.2</junit.platform.version>
-        <mockito.version>1.10.19</mockito.version>
+        <mockito.version>4.6.1</mockito.version>
     </properties>
 
     <repositories>
@@ -900,37 +900,19 @@
                 <type>pom</type>
                 <scope>import</scope>
             </dependency>
-
-            <!-- Test Dependencies -->
             <dependency>
-                <groupId>org.junit.jupiter</groupId>
-                <artifactId>junit-jupiter-engine</artifactId>
+                <groupId>org.junit</groupId>
+                <artifactId>junit-bom</artifactId>
                 <version>${junit.jupiter.version}</version>
-                <scope>test</scope>
-            </dependency>
-            <dependency>
-                <groupId>org.junit.jupiter</groupId>
-                <artifactId>junit-jupiter-api</artifactId>
-                <version>${junit.jupiter.version}</version>
-                <scope>test</scope>
-            </dependency>
-            <dependency>
-                <groupId>org.junit.jupiter</groupId>
-                <artifactId>junit-jupiter-params</artifactId>
-                <version>${junit.jupiter.version}</version>
-                <scope>test</scope>
-            </dependency>
-            <dependency>
-                <groupId>org.junit.platform</groupId>
-                <artifactId>junit-platform-suite</artifactId>
-                <version>${junit.platform.version}</version>
-                <scope>test</scope>
+                <type>pom</type>
+                <scope>import</scope>
             </dependency>
             <dependency>
                 <groupId>org.mockito</groupId>
-                <artifactId>mockito-all</artifactId>
+                <artifactId>mockito-bom</artifactId>
                 <version>${mockito.version}</version>
-                <scope>test</scope>
+                <type>pom</type>
+                <scope>import</scope>
             </dependency>
         </dependencies>
     </dependencyManagement>

--- a/pom.xml
+++ b/pom.xml
@@ -125,7 +125,7 @@
                     <configuration>
                         <java>
                             <googleJavaFormat>
-                                <version>1.8</version>
+                                <version>1.15.0</version>
                                 <style>GOOGLE</style>
                             </googleJavaFormat>
                         </java>

